<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>973</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTietokanta">
    <property name="title">
     <string>Tietokanta</string>
    </property>
    <addaction name="actionServerSettings"/>
   </widget>
   <widget class="QMenu" name="menuOhje">
    <property name="title">
     <string>Ohje</string>
    </property>
    <addaction name="actionManual"/>
    <addaction name="separator"/>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuTietokanta"/>
   <addaction name="menuOhje"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionServerSettings">
   <property name="text">
    <string>Palvelinasetukset...</string>
   </property>
   <property name="toolTip">
    <string>Muuta tietokantapalvelimen asetuksia</string>
   </property>
  </action>
  <action name="actionManual">
   <property name="text">
    <string>Ohje...</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Tietoja ohjelmasta...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
